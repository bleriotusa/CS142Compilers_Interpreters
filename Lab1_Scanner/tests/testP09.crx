int a11b = l33t - sp33k - c4n + tr1p + sc4nn3rs.
