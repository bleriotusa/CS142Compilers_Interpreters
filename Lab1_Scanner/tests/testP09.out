IDENTIFIER(int)(lineNum:1, charPos:1)
IDENTIFIER(a11b)(lineNum:1, charPos:5)
ASSIGN(lineNum:1, charPos:10)
IDENTIFIER(l33t)(lineNum:1, charPos:12)
SUB(lineNum:1, charPos:17)
IDENTIFIER(sp33k)(lineNum:1, charPos:19)
SUB(lineNum:1, charPos:25)
IDENTIFIER(c4n)(lineNum:1, charPos:27)
ADD(lineNum:1, charPos:31)
IDENTIFIER(tr1p)(lineNum:1, charPos:33)
ADD(lineNum:1, charPos:38)
IDENTIFIER(sc4nn3rs)(lineNum:1, charPos:40)
ERROR(Unexpected character: .)(lineNum:1, charPos:48)
EOF(lineNum:2, charPos:1)
